tools/xtce_generation/telemetry_generator.py

```diff
@@ -245,7 +269,9 @@ def create_container_set(
             The updated TelemetryMetaData element.
```

Comment:
You can update the docstring for this function to reflect the changes: `telemetry_metadata` is of type `Et.Element`